venv: add page (#10078)
tldr-pages/tldr: pages/common/venv.md

# venv

> > Create lightweight "virtual environments" in python.

Reviewer: ```suggestion
> Create lightweight virtual environments in python.
```

> More information: <https://docs.python.org/3/library/venv.html>.

- Create a python virtual environment:

> ` python -m venv {{path/to/new/virtual/environment}}`

Reviewer: ```suggestion
` python -m venv {{path/to/new_venv}}`
```
I think this would be better.

Author: Hi
I think `new_venv` can get confused with `venv`.
Can we keep it as `new_virtual_env` ?

Reviewer: Sure, I will request a review from other maintainers for a second opinion.

Reviewer: > Hi I think `new_venv` can get confused with `venv`. Can we keep it as `new_virtual_env` ?

```suggestion
` python -m venv {{path/to/virtual_environment}}`
```

> - Activate the virtual environment:
> 
> `source {{path/to/new/virtual/environment}}/bin/activate`

Reviewer: ```suggestion
- Activate the virtual environment (Linux and Mac os):

`source {{path/to/virtual_environment}}/bin/activate`

- Activate the virtual environment (Windows):

`{{path\to\virtual_environment}}\Scripts\activate.bat`
```

> `source {{path/to/new/virtual/environment}}/bin/activate`

Reviewer: ```suggestion
`source {{path/to/new_venv}}/bin/activate`
```

# venv

> Create lightweight virtual environments in python.
> More information: <https://docs.python.org/3/library/venv.html>.

- Create a python virtual environment:

` python -m venv {{path/to/virtual_environment}}`

> - Activate the virtual environment (Linux and Mac os):

Reviewer 2: Hello, thank you for the contribution, I will add a suggestion:

```suggestion
- Activate the virtual environment (Linux and Mac OS):
```
